src/context/WalletProvider/NewWalletViews/sections/OthersSection.tsx

```diff
@@ -44,6 +46,41 @@ const WalletConnectOption = ({
   )
 }
 
+const CoinbaseQROption = ({
+  connect,
+  isSelected,
+  isDisabled,
+}: {
+  connect: () => void
+  isSelected: boolean
+  isDisabled: boolean
+}) => {
+  const backgroundColor = useColorModeValue('blackAlpha.100', 'whiteAlpha.100')
```

Comment:
very-nitpick: `selectedBackgroundColor` whould be more descriptive

Reply:
e3db08acd1